File: apps/institution-api/Models/InstitutionAddress.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class InstitutionAddress
    {
        public int InstitutionAddressId { get; set; }
        public int InstitutionId { get; set; }
        public Guid? AddressGuid { get; set; }
        public string AddressUsage { get; set; }
        public string AddressType { get; set; }
        public int? ListSequence { get; set; }
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }
        public string AddressLine3 { get; set; }
        public string AddressLine4 { get; set; }
        public string City { get; set; }
        public string ProvinceState { get; set; }
        public string PostalZipCode { get; set; }
        public string Country { get; set; }
        public DateTime? AddressBusinessStartDate { get; set; }
        public DateTime? AddressBusinessEndDate { get; set; }

        public virtual Institution Institution { get; set; }
    }
}

File: apps/planmp/src/libs/common/models/institution-address.ts


import { Institution } from './institution';

export class InstitutionAddress {
    public institutionAddressId: number = 0;
    public institutionId: number = 0;
    public addressGuid: string;
    public addressUsage: string = '';
    public addressType: string = '';
    public listSequence: number;
    public addressLine1: string = '';
    public addressLine2: string = '';
    public addressLine3: string = '';
    public addressLine4: string = '';
    public city: string = '';
    public provinceState: string = '';
    public postalZipCode: string = '';
    public country: string = '';
    public addressBusinessStartDate: Date;
    public addressBusinessEndDate: Date;
    public institution: Institution = new Institution();

    constructor(init? : Partial<InstitutionAddress>) {
        
        Object.assign(this, init);
    }
}

